bcda/cclf/local_fileprocessor.go

```diff
@@ -123,13 +128,52 @@ func (p *processor) walk(path string, info os.FileInfo, err error) error {
 			continue
 		}
 
-		key := metadataKey{perfYear: metadata.perfYear, fileType: metadata.fileType}
-		sub := p.cclfMap[metadata.acoID]
-		if sub == nil {
-			sub = make(map[metadataKey][]*cclfFileMetadata)
-			p.cclfMap[metadata.acoID] = sub
+		if metadata.cclfNum == 0 {
+			if cclf0Metadata != nil {
```

Comment:
Where do the values come from for `cclf0Metadata` and `cclf8Metadata`?

I think I'm missing a step for the comparison here and at line 139; these values are pointers to the type `cclfFileMetadata`, per line 115. They must be getting a value somewhere or they would always be nil at this comparison?  My eyes are probably looking right at it and I'm missing it. 😅

Reply:
yes! It's right after the conditional checks on line 136 and 143 😄 

```go
cclf0Metadata = &metadata
```

Reply:
if the value is assigned after, wouldn't it always be nil? (sorry)

Reply:
This is run in a for loop which iterates over the files in the zip, so we could find a CCLF0 and then later on find another one, which would be an error :)

Reply:
that's what I wasn't seeing - thank you! 😄